src/character.h

```diff
@@ -1846,7 +1846,7 @@ class Character : public Creature, public visitable
 
         // weapon + worn (for death, etc)
         std::vector<item *> inv_dump();
-
+        std::vector<const item *> const_inv_dump() const;
```

Comment:
you can just have this be an override that's const, instead of a new named function.